common/config/config.go

```diff
@@ -44,8 +43,10 @@ type (
 		Persistence Persistence `yaml:"persistence"`
 		// Log is the logging config
 		Log Logger `yaml:"log"`
-		// ClusterMetadata is the config containing all valid clusters and active cluster
-		ClusterMetadata *ClusterMetadata `yaml:"clusterMetadata"`
+		// ClusterGroupMetadata is the config containing all valid clusters and active cluster
+		ClusterGroupMetadata *ClusterGroupMetadata `yaml:"clusterGroupMetadata"`
+		// ClusterMetadata is deprecated. Please use ClusterGroupMetadata
```

Comment:
To make it deprecated, start with `"Deprecated:" `
https://stackoverflow.com/questions/7849663/how-do-you-mark-code-as-deprecated-in-go/36360323

Reply:
Good tip! Updated.